### PR TITLE
 Add missing line for SubProcess unit test diff

### PR DESCRIPTION
**Pull request by qliphy**
#### PR description:

This fixes the unit test diff.
See the unit test error error reported in https://github.com/cms-sw/cmssw/pull/38728#issuecomment-1185588405

#### PR validation:

Test now passes.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

backport of #38754

### CODE REVIEW

**Comment by qliphy** [comment]
please test

**Comment by Dr15Jones** [comment]
Ah, you beat me to it :)

**Comment by perrotta** [comment]
@qliphy @Dr15Jones thank both you!

**Comment by perrotta** [comment]
backport of https://github.com/cms-sw/cmssw/pull/38754

**Comment by makortel** [comment]
@cmsbuild, please test

**Comment by perrotta** [comment]
@makortel I plan to merge this now anyhow, since we want to cut 12_3_4 by tomorrow, and I'd prefer having clean unit tests with it

**Comment by makortel** [comment]
+1

Ok

**Comment by perrotta** [comment]
merge
(with the intention to fix the now broken unit test in time for 12_4_3)

File: FWCore/Integration/test/unit_test_outputs/testSubProcess.grep.txt [hunk 1]
```diff
@@ -12,3 +12,4 @@ got data of type "edmtest::Doodad" with name "abcd" in record GadgetRcd
 ++++++++ finished: processing esmodule: label = '' type = DoodadESSource in record = GadgetRcd
 got data of type "edmtest::Doodad" with name "abc" in record GadgetRcd
 got data of type "edmtest::Doodad" with name "abc" in record GadgetRcd
+   25 Tracer               -s DoodadESSource:D                       4        4
```
Comment (perrotta) [inline FWCore/Integration/test/unit_test_outputs/testSubProcess.grep.txt:15]: @makortel
"25" should be a "24" in 12_4_X (whatever it means...)

Comment (qliphy) [reply FWCore/Integration/test/unit_test_outputs/testSubProcess.grep.txt:15]: @perrotta Thanks for the suggestion, it indeed works with 12_4_X.   A new fix is made: #38767 38767

**Pull request merged**